src/components/Footer.jsx

```diff
@@ -2,12 +2,12 @@ import React from "react";
 import PrimaryLink from "../styles/Typography/PrimaryLink";
 import {fontSize} from "../styles/variables";
 
-function Footer() {
+function Footer({date}) {
   return (
     <footer>
       <small style={{fontSize: fontSize.small}}>
         Made with <PrimaryLink href="https://www.onegraph.com/">OneGraph</PrimaryLink>. Hosted on{" "}
-        <PrimaryLink href="https://www.netlify.com/">Netlify</PrimaryLink>.
+        <PrimaryLink href="https://www.netlify.com/">Netlify</PrimaryLink>. Built at {date}
```

Comment:
this date could use some formatting, meybe undo the `Open Sauced vX.X.X` title and add this metadata in the <head>